### PR TITLE
Added Supported Networks 

### PR DESCRIPTION
**Pull request by vignesha22**
## Description

- Added More supported networks which have been recently deployed

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/sdk/network/constants.ts [hunk 1]
```diff
@@ -8,9 +8,19 @@ export enum NetworkNames {
   Optimism = 'optimism',
   Polygon = 'polygon',
   Arbitrum = 'arbitrum',
+  ArbitrumGoerli = 'arbitrumGoerli',
+  Chiado = 'chiado',
+  Fuse = 'fuse',
+  FuseSparknet = 'fuseSparknet',
+  Gnosis = 'gnosis',
+  Kroma = 'kroma',
```
Comment (ch4r10t33r) [inline src/sdk/network/constants.ts:16]: Kroma is testnet. Pls rename this accordingly.

File: src/sdk/network/constants.ts [hunk 2]
```diff
@@ -8,9 +8,19 @@ export enum NetworkNames {
...
+  Mainnet = 'mainnet',
+  OptimismGoerli = 'optimismGoerli',
+  Rskt = 'rskt',
```
Comment (ch4r10t33r) [inline src/sdk/network/constants.ts:19]: RSK can probably be called as RSK testnet as well?

File: src/sdk/network/constants.ts [hunk 3]
```diff
@@ -8,9 +8,19 @@ export enum NetworkNames {
...
+  Verse = 'verse',
```
Comment (ch4r10t33r) [inline src/sdk/network/constants.ts:20]: Verse is a testnet as well.

File: src/sdk/network/constants.ts [hunk 4]
```diff
@@ -118,6 +138,136 @@ export const Networks: {
       url: '',
     },
   },
+  [1]: {
+    chainId: 1,
+    bundler: 'https://ethereum-bundler.etherspot.io/',
+    contracts: {
+      entryPoint: '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789',
+      walletFactory: '0x2f771DCa6Ffa3879e48355E8A4aF5b81d82A6164',
+      uniswapV3SwapRouter: '0xE592427A0AEce92De3Edee1F18E0157C05861564',
```
Comment (ch4r10t33r) [inline src/sdk/network/constants.ts:147]: Do we need to store uniswapV3Router here? Can we move it to a separate constants file? Let's retain this only for our contracts?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

File: src/sdk/network/constants.ts (hunk 4 above)
Comment (vignesha22) [reply src/sdk/network/constants.ts:147]: actually we don't need to store the uniswap router since we don't use it anywhere

**Review by ch4r10t33r — APPROVED** [review]
LGTM

**Review by marie-fourier — APPROVED** [review]

**Pull request merged**